cspell.json

```diff
@@ -297,11 +297,13 @@
         "cryptoartai",
         "csrftoken",
         "cusd",
+        "dczs",
         "defi",
         "deletable",
         "devnet",
         "dodoex",
         "drawed",
+        "eaaaaa",
```

Comment:
This word looks not right to me. FYI, you can also use `/* cspell:disable-next-line */` to silent the cspell.